langchain/vectorstores/zilliz.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import annotations
+
+import logging
+from typing import Any, List, Optional
+
+from langchain.embeddings.base import Embeddings
+from langchain.schema import Document
+from langchain.vectorstores.milvus import Milvus
+
+logger = logging.getLogger(__name__)
+
+
+class Zilliz(Milvus):
+    def _create_index(self) -> None:
+        """Create a index on the collection"""
+        from pymilvus import Collection, MilvusException
+
+        if type(self.col) == Collection and self._get_index() is None:
```

Comment:
would `isinstance(self.col, Collection)` be better than direct equality check

Reply:
True, in the current cases it wont have to check for inheritance but might be worth it to just add the extra safety.